<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitCommandsForm</class>
 <widget class="QWidget" name="GitCommandsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Here the git history is shown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="container">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="branchListCombo"/>
     </item>
     <item>
      <widget class="QToolButton" name="refreshBranchesButton">
       <property name="toolTip">
        <string>Reload branches</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="checkoutBranchButton">
       <property name="toolTip">
        <string>Checkout the selected branch</string>
       </property>
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="diffBranchButton">
       <property name="toolTip">
        <string>diff current branch against selected branch</string>
       </property>
       <property name="text">
        <string>Diff...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newBranchButton">
       <property name="toolTip">
        <string>Create a new local branch</string>
       </property>
       <property name="text">
        <string>New branch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
